tests/shared/src/test/scala-2.13+/cats/tests/NonEmptyLazyListSuite.scala

```diff
@@ -49,6 +49,7 @@ class NonEmptyLazyListSuite extends NonEmptyCollectionSuite[LazyList, NonEmptyLa
   checkAll(s"NonEmptyLazyList[Int]", HashTests[NonEmptyLazyList[Int]].hash)
   checkAll(s"Hash[NonEmptyLazyList[Int]]", SerializableTests.serializable(Hash[NonEmptyLazyList[Int]]))
 
+  checkAll("NonEmptyLazyList[Int]", NonEmptyAlternativeTests[NonEmptyLazyList].nonEmptyAlternative[Int, Int, Int])
```

Comment:
This statement seems duplicating the next one – is it a typo or am I missing something?

Reply:
Yikes. Thanks for spotting this 🙌